server/auth/providers/google.js

```diff
@@ -87,7 +87,13 @@ if (GOOGLE_CLIENT_ID) {
     )
   );
 
-  router.get("google", passport.authenticate(providerName));
+  router.get(
+    "google",
+    passport.authenticate(providerName, {
+      accessType: "offline",
+      prompt: "consent",
```

Comment:
Does this override L39?

https://github.com/outline/outline/blob/09623b4050e2fd51b6a914fbca3f90e2ae28ac91/server/auth/providers/google.js#L39

Reply:
It does override.